### ISSUE TITLE
Add link data type

### ISSUE THREAD

**Issue by kuschanton** [danbartlett-twilio/twilio-table#1]
I had to modify the code in a bit hacky way in order to achieve it, but would be good to be able to add links to the table or at least make it recognise URLs and format them as HTML links
![Screenshot 2022-08-04 at 18 05 19](https://user-images.githubusercontent.com/4056707/182894812-886458e3-4fdf-423b-9726-23d55b2503e8.png)
.

**Comment by danbartlett-twilio** [comment]
Good idea! I just committed code that should work if you want to give it a try...
